tests/test_string_validator.py

```diff
@@ -1,6 +1,7 @@
+import sys
+import os
+sys.path.append(os.getcwd())
```

Comment:
This doesn't seem necessary.

Reply:
Receive the following error when running tests/test_custom_checker.py with
`from pyQuARC.code.string_validator import StringValidator`
```
Traceback (most recent call last):
  File "/Users/sam/Documents/GitHub/pyQuARC/tests/test_string_validator.py", line 1, in <module>
    from pyQuARC.code.string_validator import StringValidator
ModuleNotFoundError: No module named 'pyQuARC'

```